modules/ROOT/pages/contribute.adoc

```diff
@@ -7,15 +7,12 @@ Many of our documentation repositories are open and we welcome your contribution
 
 To contribute to the English version of this documentation:
 
-. Select *Edit in Github* on the page you want to modify.
+. If this is your first time contributing, sign the https://cla.salesforce.com/sign-cla[Salesforce.com Contributor License Agreement (CLA)]. The CLA only needs to be signed once per GitHub account.
+. Click *Edit in Github* on the page you want to modify.
+. Make your edits.
 . Submit a pull request to the Git repository.
-. If this is the first time you are submitting a pull request,
-run the https://api-notebook.anypoint.mulesoft.com/notebooks/#d28e8f95e739ee1e1e30d5ec577f4a27[API Notebook].
-If you aren't already logged in to your GitHub account, you can do so using a link provided in the API Notebook.
 
-image:api-notebook1.png[User interface for running the API notebook to become a contributor]
+If you submit the pull request without signing the CLA first, you will receive an error in the pull request. Sign the CLA to remove this error.
 
-If you submit the pull request without running the API Notebook first, you will receive a message in the pull request asking you to do so.
-The API Notebook only needs to run once per GitHub account.
 
-The Documentation team will either accept your PR or reach out if there are questions.
\ No newline at end of file
+The Documentation team will either accept your pull request or reach out if there are questions.
```

Comment:
We try to avoid verb tenses other than present.
```suggestion
The Documentation team either accepts your pull request or contacts you for clarification.
```
We also try to avoid idioms, like "reach out," just to make things a bit easier for translation.
One more thing to avoid is "there is" and "there are" when they act as pseudo-subjects; see Style Ref.